packages/terra-form-select/src/multiple/Menu.jsx

```diff
@@ -341,40 +338,25 @@ class Menu extends React.Component {
 
       const option = MenuUtil.findByValue(children, active);
       // Handles communicating the case where a regular option is selected to screen readers.
-      /*
-        Detecting if browser is not Safari before updating aria-live as there is some odd behaivor
-        with VoiceOver on desktop, that causes the selected option to be read twice when this is
-        is added to aria-live container.
-        When we shift focus back to select, VoiceOver automatically reads the display text.
-        Using aria-hidden on the display does not prevent VO from reading the display text and so it
-        results in reading the display text followed by reading the aria-live message which is
-        the display text + 'selected'
-        */
-      if (SharedUtil.isSafari()) {
```

Comment:
removed this check.. leftover from the refactor (missed it then)

doesn't appear to make a difference, especially since both blocks run the same thing